src/System.Drawing.Common/src/Resources/Strings.resx

```diff
@@ -343,4 +402,10 @@
   <data name="BlendObjectLastElementInvalid" xml:space="preserve">
     <value>Invalid Blend object. The positions array must have 1.0 as its last element.</value>
   </data>
+  <data name="DllNotFoundExceptionMessage" xml:space="preserve">
+    <value>Unable to load DLL '{0}': The specified module could not be found.</value>
+  </data>
+  <data name="EntryPointNotFoundExceptionMessage" xml:space="preserve">
+    <value>Unable to find an entry point named '{0}' in DLL '{1}'.</value>
+  </data>
```

Comment:
do you need to remove this as dead? maybe I'm missing where it's used
```xml
  <data name="LibgdiplusNotFound" xml:space="preserve">
    <value>The native library "libgdiplus" is not installed on the system, or was otherwise unable to be loaded.</value>
  </data>
```